README.md

```diff
@@ -1,63 +1,87 @@
+![PRs welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)
+![License](https://img.shields.io/npm/l/effector.svg?colorB=brightgreen&style=popout)
 
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-## Table of Contents
+## Introduction
 
-- [Motivation](#motivation)
-- [Installation](#installation)
-- [About](#about)
-  - [Hello world with events and nodejs](#hello-world-with-events-and-nodejs)
-  - [Storages and events](#storages-and-events)
-- [Demo](#demo)
-- [Wiki](#wiki)
-- [Usage](#usage)
-  - [Domain hooks](#domain-hooks)
-- [Typings](#typings)
-- [API](#api)
-- [Core types](#core-types)
-- [Contributors](#contributors)
-- [License](#license)
+Effector is an effective multi store state manager for Javascript apps __(React/Vue/Node.js)__, that allows you to manage data in complex applications without the risk of inflating the monolithic central store, with clear control flow, good support types and high capacity API. Effector supports both TypeScript and Flow type annotations out of the box.
 
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+> Detailed comparison with other state managers you can be found [here]()
 
-
-## Motivation
-
-At the start (early 2018) we had a lot of state-managers, those solve actual problems, but not elegant and with some limitations.
-
-Effector tries to solve all [state-manager's] user problems and most parts of limitations. Primary:
-- [x] Developer-friendly, [static] type safe and inference, boilerplate free, insinuating API.
-- [x] Maximum update (state and subscribers) performance  or minimum library performance cost by compile time / initial runtime computation.
-- [x] Modular, isomorphic, flexible.
-- [x] Tiny bundle size.
+### Effector follows five basic principles:
+- __Application stores should be as light as possible__ - the idea of adding a store for specific needs should not be frightening or damaging to the developer. Stores should be freely combined - the idea is that the data that an application needs can be distributed statically, showing how it will be converted during application operation.
+- __Application stores should be freely combined__ - data that the application needs can be statically distributed, showing how it will be converted in runtime.
+- __Autonomy from controversial concepts__ - no decorators, no need to use classes or proxies - this is not required to control the state of the application and therefore the api library uses only functions and simple js objects
+- __Predictability and clarity of API__ - A small number of basic principles are reused in different cases, reducing the user's workload and increasing recognition. For example, knowing how .watch works in events, you can guess how does function .watch in store.
```

Comment:
Better phrasing:

*For example, if you know how `.watch` works for events, you already know how `.watch` works for stores.*